pkg/web/ui/components/quote_display.templ

```diff
@@ -29,8 +29,8 @@ templ quoteContent(props QuoteDisplayProps) {
 
 templ QuoteDisplay(props QuoteDisplayProps) {
 	<div class={ "flex flex-col", props.ContainerClass }>
-		<div class={ "mb-[-0.5rem] w-fit rounded-t-lg bg-zinc-700 p-[0_0.5rem_0.5rem_0.5rem] text-center md:block", templ.KV("hidden", props.AsButton) }>
-			{ props.Label }
+		<div class={ "mb-[-0.5rem] w-fit rounded-t-lg bg-zinc-700 p-[0_0.5rem_0.5rem_0.5rem] text-center md:flex flex-row gap-1", templ.KV("hidden", props.AsButton) }>
```

Comment:
Can you sort these classes? In VS Code, with the tailwind extension installed, select the text between the quotes and do Cmd/Ctrl-Shift-P -> `Tailwind CSS: Sort Selection`